db/migrate/20230128041011_create_checks.rb

```diff
@@ -1,7 +1,7 @@
 class CreateChecks < ActiveRecord::Migration[7.0]
   def change
     create_table :checks do |t|
-      t.belongs_to :employee, index: true, foreign_key: true
+      t.belongs_to :employee, index: true, null: false, foreign_key: true
```

Comment:
You shouldn't modify an existing migration instead create a new one

Reply:
We create a new migration but with the same name, so maybe GitHub recognize like a modified file